gradle-baseline-java/build.gradle

```diff
@@ -10,7 +10,7 @@ dependencies {
     compile 'com.diffplug.spotless:spotless-plugin-gradle'
     compile 'com.google.errorprone:error_prone_refaster'
     compile 'com.google.guava:guava'
-    compile 'com.netflix.nebula:nebula-dependency-recommender'
+    compileOnly 'com.netflix.nebula:nebula-dependency-recommender'
```

Comment:
you may want to update the readme so that users know to bring their own nebula-dependency-recommender